### PR TITLE
Project News Site - Tina

### PR DESCRIPTION
**Pull request by TinaJuren**
I would appreciate feedback on the overall structure of the code.

1. What's unclear?
2. What should be added?
3. What should be removed?

Thank you / Tina

### CODE REVIEW

**Review by HIPPIEKICK — APPROVED** [review]
Good job with this project Tina! The responsiveness is good and the structure is good. Just remember that you don't have to give all elements different class names if you're not styling them differently.

We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/index.html [hunk 1]
```diff
@@ -4,19 +4,114 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/index.html]: Thank you, it's changed :)

File: code/index.html [hunk 2]
```diff
@@ -4,19 +4,114 @@
...
+        <a class="navigation-item" href="#"> Contact</a>
+      </nav>
     </header>
+    
+    <section class="welcome-news welcome-container">
+      <article class="welcome-card">
+        <img class="welcome-image" src="welcome-image.jpg" alt="Childs drawing of a tree">
+        <div class="welcome-text">
+          <h1>The Five AM Story Features - Wordpoems</h1>
+          <p><em>- one poem each day for a hundred days to remain sane</em></p>
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/index.html]: Great suggestion :)

File: code/index.html [hunk 3]
```diff
@@ -4,19 +4,114 @@
...
+          <p>Requirements for each poem:<p> 
+          <ul>
+            <li>include Merriam Websters Word of the Day</li>
+            <li>be published on that day</li>
+            <li>Bonus: include a photo</li>
+          </ul>
+          <p>Below you can find eight of the Wordpoems.</p>
+          <p><a class="link" href="#">View the whole collection here.</a><p>
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/index.html]: No :) I removed it now.

File: code/index.html [hunk 4]
```diff
@@ -4,19 +4,114 @@
...
+      <img class="logo" src="logo-thefiveamstory-210120.png"/>
+      <nav class="navigation">
+        <a class="navigation-item" href="#">Poems</a>
+        <a class="navigation-item" href="#">About</a>
+        <a class="navigation-item" href="#"> Contact</a>
+      </nav>
     </header>
+    
+    <section class="welcome-news welcome-container">
+      <article class="welcome-card">
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/index.html:22]: Thank you :)

File: code/index.html [hunk 5]
```diff
@@ -4,19 +4,114 @@
...
+      </article> 
+
+      <article class="poem-card poem-card-two">
+        <img class="poem-image" src="poem-image-2.jpg" alt="Cow with twin-calves">
+        <div class="poem-text">
+            <h2>Tincture of dawn</h2>
+            <p>January 30, 2018 – Tincture – a medicinal substance or a trace or tint</p>
+            <a class="link" href="http://thefiveamstory.com/2018/01/30/tincture-of-dawn/">Read the whole poem</a>
+        </div>
+      </article>
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/index.html]: Thanks, you're right they don't :)

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,325 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  font-size: 16px;
+}
+
+.container {
+  width: 320px;
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/style.css]: Yes, I changes the width to 100%, thanks :)

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,325 @@
...
+  width: 80%;
+  height: 500px;
+  margin-top: 30px;
+  border-radius: 20px;
+  background: #fff;
+  position: relative;
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.1);
+  -webkit-transition: all 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  transition: all 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  }
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/style.css:148]: Thanks. I wanted googled around for inspiration :)

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,325 @@
...
+  opacity: 0;
+  -webkit-transition: all 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  transition: all 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  }
+
+/* Scale up the poem-card */
+.poem-card:hover {
+  -webkit-transform: scale(1.25, 1.25);
+  transform: scale(1.25, 1.25);
+}
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/style.css]: Thanks for the heads up! I added z-index 2 and now it looks better, and I got to practice :)

File: code/style.css [hunk 9]
```diff
@@ -1,4 +1,325 @@
...
+
+/* Fade in the pseudo-element with the bigger shadow */
+.poem-card:hover::after {
+  opacity: 1;
+}
```
[the remark this answers is not in the document]
Comment (TinaJuren) [reply code/style.css]: Me neither :), I removed it. Thanks!

**Pull request closed without merging**